src/features/onboarding/components/OnboardingPopup.tsx

```diff
@@ -14,14 +14,17 @@ interface Props {
 export const OnboardingPopup: FC<Props> = ({ children }) => {
   const { openOnboarding } = useContext(OnboardingContext);
   const isDefaultState = useSelector<RootState>(selectIsDefaultState);
+  const { isOpen: isOnboardingOpen } = useContext(OnboardingContext);
+
   const isOnboardingAvailable =
-    useSelector<RootState>((_state) => _state.main.backendAvailable) === BackendStatus.Accessed;
+    useSelector<RootState>((_state) => _state.main.backendAvailable) === BackendStatus.Accessed && !isOnboardingOpen;
```

Comment:
probably, it would be useful to add this constant to context and reuse

Reply:
Good idea, moved to context